cdc/entry/mounter.go

```diff
@@ -555,6 +556,13 @@ func formatColVal(datum types.Datum, tp byte) (interface{}, error) {
 			b = emptyBytes
 		}
 		return b, nil
+	case mysql.TypeFloat, mysql.TypeDouble:
+		v := datum.GetFloat64()
+		if math.IsNaN(v) || math.IsInf(v, 1) || math.IsInf(v, -1) {
+			v = 0
+			log.Warn("the value is invalid in column", zap.Float64("value", v))
```

Comment:
Oh, seems the log doesn't help user to locate the row.
Btw we should log before `v` is set to zero